sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
 
+## 1.9.0 (2023-05-04)
+
+### Features Added
+
+- Added the ability to ignore invalid certificate common name for TLS connections in WinHTTP transport.
+- Added `DisableTlsCertificateValidation` in `TransportOptions`.
+- Added `TokenCredential::GetCredentialName()` to be utilized in diagnostic messages. If you have any custom implementations of `TokenCredential`, it is recommended to pass the name of your credential to `TokenCredential` constructor. The old parameterless constructor is deprecated.
+- Added support for challenge-based and multi-tenant authentication.
```

Comment:
I think we should consider keeping this in beta until next month, or at least after an api review is done.

We should close on that discussion.

Reply:
I replied in two comments below.